lib/repos/likes_repo.dart

```diff
@@ -13,7 +13,11 @@ class LikesRepo {
 
   Future<int> getNumberOfLikesByTileId(String tileId) async {
     var likes = await likesDao.getLikesByTileId(tileId);
-    return likes.length;
+    if (likes == null) {
```

Comment:
You can also do this in a compact way like this:
likes?.length ?? 0

Reply:
Okay, sir, I will update it.